api/disruption_types_test.go

```diff
@@ -22,26 +23,29 @@ var _ = Describe("DisruptionStatus.RemoveDeadTargets Test", func() {
 	BeforeEach(func() {
 		rand.Seed(time.Now().UnixNano())
 		status = &v1beta1.DisruptionStatus{
-			Targets: []string{"0.0.0.0", "1.1.1.1", "2.2.2.2", "3.3.3.3", "4.4.4.4"},
+			TargetInjections: *makeValidTargetInjections(),
 		}
 	})
 
-	JustBeforeEach(func() {
-		rand.Shuffle(len(matchingTargets), func(i, j int) {
-			matchingTargets[i], matchingTargets[j] = matchingTargets[j], matchingTargets[i]
-		})
-	})
-
 	When("matchingTargets contains some current targets", func() {
+		var saveTargets v1beta1.TargetInjections
+
 		BeforeEach(func() {
-			matchingTargets = status.Targets
+			matchingTargets = []string{
+				"0.0.0.0",
```

Comment:
I'm a little confused, are those supposed to be names or IPs? I believe it's the associated target name (either pod or node), maybe making those tests values more representative would be a plus (if my supposition is correct)